package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "gamepad-mappings",
+  "version": "1.0.0",
```

Comment:
Is this at a functional-enough state to be considered version `1.0.0`? Don't really care what the version is, honestly, but people tend to think that "1.0" is a statement of stability/release-worthiness.

Reply:
Ah typo. I'll fix it.  I meant to say 0.1.0